tests/components/accessibilityControls-test.js

```diff
@@ -133,6 +134,147 @@ describe('AccessibilityControls', function () {
+      });
+
+    });
+
+    describe('changeVolumeBy', function() {
+
+      it('should increase and decrease the volume', function() {
+        var volume = 0;
+        mockCtrl.setVolume = function(vol) {
+          volume = vol;
+        };
+        mockCtrl.state.volumeState.volume = 0;
+        a11yCtrls.changeVolumeBy(100, true);
+        expect(volume).toBe(1);
+        mockCtrl.state.volumeState.volume = 1;
+        a11yCtrls.changeVolumeBy(100, false);
+        expect(volume).toBe(0);
+        mockCtrl.state.volumeState.volume = 0.5;
+        a11yCtrls.changeVolumeBy(25, true);
+        expect(volume).toBe(0.75);
+      });
+
+      it('should not call controller.setVolume() when requested change results in current volume', function() {
+        var setVolumeCalled = false;
+        mockCtrl.setVolume = function() {
+          setVolumeCalled = true;
+        };
+        mockCtrl.state.volumeState.volume = 0;
+        a11yCtrls.changeVolumeBy(100, false);
+        expect(setVolumeCalled).toBe(false);
+      });
+
+      it('should constrain volume to supported values', function() {
+        var volume = 0;
+        mockCtrl.setVolume = function(vol) {
+          volume = vol;
+        };
+        mockCtrl.state.volumeState.volume = 0.5;
+        a11yCtrls.changeVolumeBy(500, true);
+        expect(volume).toBe(1);
+        a11yCtrls.changeVolumeBy(500, false);
+        expect(volume).toBe(0);
+      });
+
+    });
+
+    describe('seekBy', function() {
+      var newPlayhead;
+
+      beforeEach(function() {
+        newPlayhead = null;
+        mockCtrl.seek = function(seekTo) {
+          newPlayhead = seekTo;
+        };
+      });
+
+      it('should seek the specified amount of seconds forward', function() {
+        mockCtrl.skin.state.duration = 60;
+        mockCtrl.skin.state.currentPlayhead = 0;
+        a11yCtrls.seekBy(5, true);
+        expect(newPlayhead).toBe(5);
+      });
+
+      it('should seek back the specified amount of seconds', function() {
+        mockCtrl.skin.state.duration = 60;
+        mockCtrl.skin.state.currentPlayhead = 10;
+        a11yCtrls.seekBy(5, false);
+        expect(newPlayhead).toBe(5);
+      });
+
+      it('should not seek past the video\'s duration', function() {
+        mockCtrl.skin.state.duration = 60;
+        mockCtrl.skin.state.currentPlayhead = 10;
+        a11yCtrls.seekBy(120, true);
+        expect(newPlayhead).toBe(mockCtrl.skin.state.duration);
+      });
+
+      it('should not seek past the video\'s start time', function() {
+        mockCtrl.skin.state.duration = 60;
+        mockCtrl.skin.state.currentPlayhead = 30;
+        a11yCtrls.seekBy(120, false);
+        expect(newPlayhead).toBe(0);
+      });
+
+      it('should not seek when seeking is disabled', function() {
+        mockCtrl.state.screenToShow = CONSTANTS.SCREEN.AD_SCREEN;
+        mockCtrl.skin.state.duration = 60;
+        mockCtrl.skin.state.currentPlayhead = 0;
+        a11yCtrls.seekBy(5, true);
```

Comment:
Can we add a test to check that negative values are ignored?

Reply:
Hmm this actually inverts the direction of the seek, but works fine. I'm trying to think what would be the proper way to react to negative values. Any thoughts?

Reply:
That sounds good to me. Can we add a test for this? Can we also check the same thing for changeVolumeBy if it functions similarly (negative inputs invert the direction)? Thanks!

Reply:
Sure, will add it

Reply:
@aeng7 changeVolumeBy works a little differently. Since that parameter is a percentage it will just constrain it to 0 percent. I will add this case to the tests